pandunia-lekse.csv

```diff
@@ -3352,6 +3410,7 @@ zone|||ell:ζώνη (zónē), eng:fra:zone, spa:por:zona, rus:зона (zona)|b
 zonga||||impale (stab, skewer)||atravesar|||||刺す||||||||||
 zonge|||zho:樁 (zhuāng), yue:樁 (zong1), nan:樁 (zuang1), jpn:樁 (shō), vie:thưởng|post (stake, peg, skewer)|pieu (poteau)|poste (estaca, palo)|poste (estaca)|кол||桩|杭||||||||||
 zukame||||cold (illness)||resfriado (catarro)||||||||||||||vilustuminen|przeziębienie, katar
+zukine|||ita:eng:zucchini, rus:цукини (tsukini), jpn:ズッキーニ (zukkīni), may:zukin|zucchini|courgette|calabacín|abobrinha|цукини||西葫芦|ズッキーニ||||||||||
```

Comment:
Is it worth of its own root from botanical point of view?

Reply:
Not really, since it's just a variety of summer squash. I was on the fence with this one, though, because "zukine" is pretty international. It can totally be "lil(i) kaboge" instead, though.

Reply:
OK. Do as you see best.

Reply:
I set it to "lilkaboge" to match with the other species names, which I also changed to compound words. it works well, actually, because "lil-" is the closest thing Pandunia has to a diminuitive, and "zuchinni" is just the Italian diminuitive of "squash".